.devcontainer/devcontainer.json

```diff
@@ -1,20 +1,20 @@
 {
     "portsAttributes": {
-        "8080": {
+        "3000": {
```

Comment:
Swapped to `serve` over `http-serve` below because I couldn't get `http-serve` to stop sending `ETags`, which meant it wasn't picking up my JS changes on reload.